scripts/audit_locg_reading_orders.py

```
@@ -0,0 +1,398 @@
+        try:
+            result = audit_list(locg_list, threads, token, issue_cache, dep_cache)
+            if result is None:
+                print("skip (< 2 owned series)")
+                skipped += 1
+            else:
+                print(
+                    f"OK ({len(result['owned_series'])} owned series, "
+                    f"{len(result['present'])}P/{len(result['missing'])}M/{len(result['not_found'])}NF)"
+                )
+                results.append(result)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)